src/components/ProfileForm/index.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import { View } from 'react-native';
+import { KeyboardAwareScrollView } from 'react-native-keyboard-aware-scroll-view';
+
+import Input from 'components/common/Input';
+import SelectAvatar from 'components/SelectAvatar';
+import strings from 'locale';
+import { PROFILE_SHAPE } from 'constants/shapes';
+import styles from './styles';
+
+const {
+  PROFILE: { usernameLabel, firstNameLabel, lastNameLabel },
+} = strings;
+
+const ProfileForm = ({ profile }) => {
+  const [avatarData, setAvatarData] = useState('');
+  const { username, firstName, lastName, email } = profile;
+  return (
+    <View style={styles.container}>
+      <KeyboardAwareScrollView keyboardShouldPersistTaps="handled">
+        <Input label={usernameLabel} value={username} additionalStyles={styles.inputContainer} />
+        <Input
+          label={strings.SIGN_UP.email}
+          keyboardType="email-address"
+          autoCapitalize="none"
+          additionalStyles={styles.inputContainer}
+          value={email}
```

Comment:
Small small comment but, it's clean to maintain the order of the props. On the other `Input` tags, you pass `value` as second parameter and then `additionalStyles`, etc.  I think it's good to maintain consistency, (or maybe I'm just crazy with details haha 😅)

Reply:
Agree! Thanks! Anyway, with the update feature addition, this doesn't look the same